addons/base_import/static/src/import_action/import_action.js

```diff
@@ -50,6 +51,28 @@ export class ImportAction extends Component {
             previewError: "",
         });
 
+        this.uploadFiles = useFileUploader();
+        useDropzone(useRef("root"), async event => {
+            const file = Array.from(event.dataTransfer.files).find(file => {
```

Comment:
If the user drops multiple files, I prefer picking the first valid file instead of showing an error. It makes the system more flexible. I can change that as well if you prefer an error :smiley:

Reply:
Meh, I prefer a clear error message, people may drag & drop 2 import files and think the system is totally fine with that, which is not the case.